ykman/cli/opgp.py

```diff
@@ -97,6 +97,10 @@ def info(ctx):
     """
     controller = ctx.obj['controller']
     click.echo('OpenPGP version: %d.%d.%d' % controller.version)
+    retries = controller.get_remaining_pin_tries()
+    click.echo('PIN tries remaining: {}'.format(retries[0]))
+    click.echo('Reset code tries remaining: {}'.format(retries[1]))
+    click.echo('Admin PIN tries remaining: {}'.format(retries[2]))
```

Comment:
I feel like we should name these values instead of addressing them by index, where possible. How about renaming `get_remaining_pin_tries()` to `_get_remaining_pin_tries()`, and adding a new `get_remaining_pin_tries()` which returns a dict instead? (I can push a prototype of that)

Reply:
See 4f77e92d8225365538e861b7100c21f214713ff8

Reply:
Agreed.

Reply:
How about a namedtuple?

Reply:
@dainnilsson Even better!